source/MRMesh/MRMatrix4.h

```diff
@@ -58,9 +58,14 @@ struct Matrix4
     /// compute sum of squared matrix elements
     constexpr T normSq() const noexcept { return x.lengthSq() + y.lengthSq() + z.lengthSq() + w.lengthSq(); }
     constexpr T norm() const noexcept { return std::sqrt( normSq() ); }
+    /// computes submatrix
+    constexpr Matrix3<T> submatrix( int i, int j ) const noexcept;
```

Comment:
better name `submatrix3` and explain what is `i` and `j` in the comment

Reply:
Fixed.